driver/src/main/java/org/neo4j/driver/internal/logging/DevNullLogger.java

```diff
@@ -19,11 +19,19 @@
  */
 package org.neo4j.driver.internal.logging;
 
+import java.util.logging.Level;
+
 import org.neo4j.driver.internal.spi.Logger;
 
 public class DevNullLogger implements Logger
 {
     @Override
+    public void log( Level level, String message )
```

Comment:
I'm not a fan of exposing JUL log level through this interface, the intent was to hide JUL behind it, so that we can plug in other log frameworks and testing if we like. Not a biggie, but perhaps we can follow up and add a `trace` method or something, rather than expose programmatic log level here.